.github/workflows/ci.yaml

```diff
@@ -94,11 +94,11 @@ jobs:
         uses: manusa/actions-setup-minikube@v2.7.2
         if: steps.list-changed.outputs.changed == 'true'
         with:
-          minikube version: v1.27.1
+          minikube version: v1.30.1
           kubernetes version: ${{ matrix.kubernetes-version }}
-          container runtime: ${{ matrix.cri }}
           driver: docker
           github token: ${{ secrets.GITHUB_TOKEN }}
+          start args: "--nodes=2 --container-runtime=${{ matrix.cri }}"
```

Comment:
Why do you need two nodes? I understand you want to specify the container runtime, but two nodes means it will take longer to spin up cluster

Reply:
I followed the fix that Jerry implemented here for the same issue: https://github.com/newrelic/nri-kube-events/pull/275

Reply:
We will need to fix that too... eventually. He was fixing a container runtime issue and added an unnecessary step. Please remove the two node requirement to avoid increasing workflow runtime

Reply:
That fix is shady. Adding `--nodes=2` is not relevant to the problem that needs to be solved. Could you please test whether workflow runs without the added flag?

Reply:
If this works, we will have to fix that in `nri-kube-events`. I will take care of that once you confirm flag is unnecessary